Android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Fri Jan 29 10:30:35 EST 2021
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
```

Comment:
Why not the latest Gradle v8.5?

Reply:
This is the minimum required version without dependency changes for now, `8.5` might introduce more breaking changes. We can upgrade that with all dependencies later.